### ISSUE TITLE
Go to definition does not work for functions inside objects

### ISSUE THREAD

**Issue by roonyh** [ballerina-platform/ballerina-lang#13764]
**Description:**
Vscode extension's Go to Definition feature does not work for functions defined inside objects and services

**Steps to reproduce:**
~~~ballerina
import ballerina/http;

function topLevel() {
}

type Test object {
    function inObject() {}
};

service serviceName on new http:Listener(8080) {
    function inService() {
    }

    resource function newResource(http:Caller caller, http:Request request) {
        topLevel(); // <- Go to definition works

        Test t = new;
        t.inObject();   // <- Go to def does not work

        self.inService();  // <- Go to definition does not work
    }
}
~~~

**Affected Versions:**
0.990.3

**Comment by nadeeshaan** [comment]
This is being addressed with the latest revamp #13529

**Comment by nadeeshaan** [comment]
Closing since the fix has been merged
